packages.go

```diff
@@ -0,0 +1,154 @@
+//
+// Copyright 2021, Kordian Bruck
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"fmt"
+	"time"
+)
+
+// PackagesService handles communication with the packages related methods
+// of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/packages.html
+type PackagesService struct {
+	client *Client
+}
+
+// Package represents a GitLab single package
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/packages.html
+type Package struct {
+	ID        int        `json:"id"`
+	Name      string     `json:"name"`
+	Version   string     `json:"version"`
+	Type      string     `json:"package_type"`
```

Comment:
I see the repetition, but lets use the actual names.

```suggestion
	PackageType      string     `json:"package_type"`
```